feathr_project/setup.py

```diff
@@ -84,6 +84,7 @@
         "azure-storage-file-datalake<=12.5.0",
         "azure-synapse-spark<=0.7.0",
         # fixing Azure Machine Learning authentication issue per https://stackoverflow.com/a/72262694/3193073
+        "aiohttp==3.8.3",  # pin to resolve Synapse import issue
```

Comment:
Can you elaborate a bit more on this? What is the specific issue?

Reply:
This is what I found from our synapse example notebook:

>If you meet errors like 'cannot import FeatherClient from feathr', it may be caused by incompatible version of 'aiohttp'. Please try to install/upgrade it by running: '%pip install aiohttp==3.8.3'

and that actually what I encountered when I tried the feathr package import on the synapse. I found pinning `aiohttp==3.8.3` resolves the issue.

Reply:
specifically, the message is `function() argument 'code' must be code, not str` throwing from aiohttp package that is installed on Synapse (version == 3.7.4.post0). I'll add this detail to the setup.py file.